packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -1140,6 +1140,11 @@ export default class RestApi {
       }
     })
 
+    api.get(
+      '/',
+      wrap((req, res) => sendObjects(Object.values(collections), req, res))
```

Comment:
I think we should add one link to the collections : docs or open api. That way even the open api is discoverable
What do you think about it?